### PR TITLE
TFAC-537 Generalizing tab-homepage Landing Section

### PR DESCRIPTION
**Pull request by jedtan**

### CODE REVIEW

**Comment by amaliwanag2** [comment]
Screen Shot 2021-11-09 at 1 37 52 PM
@jedtan so I realized I had a bug in my original code that didn't really show up until I added the footer section. 

The wave SVG snaps to the bottom of the screen on mobile and is positioned absolute, but we actually want it to still show at just the bottom of the landing page section.  Can you adjust so that

File: src/components/Landing.js [hunk 1]
```diff
@@ -0,0 +1,272 @@
...
+            onUnsupportedBrowserInstallClick={() => {
+              setShowUnsupportedBrowserMessage(true)
+            }}
+          />
+        </div>
+        <div className={cx.halfScreenRight}>
+          <img src={headerImg}></img>
+        </div>
+        <div className={cx.wave}>
+          <Wave />
```
Comment (amaliwanag2) [inline src/components/Landing.js:223]: let's make the wave component take a hex value as a prop and generate the svg fill color based on the prop

File: src/components/Landing.js [hunk 2]
```diff
@@ -0,0 +1,272 @@
...
+    localStorageMgr.setItem(STORAGE_NEW_USER_CAUSE_ID, causeId)
+  }
+  const absolutePageURL = getAbsoluteURL(location.pathname)
+  let headerImg
+  switch (causeId) {
+    case STORAGE_SEAS_CAUSE_ID:
+      headerImg = seasHeaderImg
+      break
+    case STORAGE_CATS_CAUSE_ID:
+      headerImg = catsHeaderImg
```
Comment (amaliwanag2) [inline src/components/Landing.js:159]: it looks like the cats header image didn't actually export the cat from figma.  You'll want to re-add the cats Header image

File: src/components/Landing.js [hunk 3]
```diff
@@ -0,0 +1,272 @@
...
+    headTitle: PropTypes.string,
+    headTitleTemplate: PropTypes.string,
+    headOgTitle: PropTypes.string,
+    headOgDescription: PropTypes.string,
+    headKeywords: PropTypes.string,
+    headOgImgURLAbsolute: PropTypes.string,
+  }),
+}
+const LandingWithTheme = (props) => (
+  <ThemeProvider theme={props.theme}>
```
Comment (amaliwanag2) [inline src/components/Landing.js:261]: lets take the ThemProvider and CssBaseline out of the component and put it in the parent component, so we don't have to define it in each section of the homepage

**Review by amaliwanag2 — CHANGES_REQUESTED** [review]
feel free to merge in my branch
alec/TFAC-694
to get the theming on your stories

File: src/components/Landing.js (hunk 3 above)
Comment (jedtan) [reply src/components/Landing.js:261]: I will probably wait for your PR to land before doing this, I'll address the other comments first.

File: gatsby-node.esm.js [hunk 4]
```diff
@@ -180,6 +180,7 @@ exports.createPages = async ({ actions, graphql }) => {
     `)
     dynamicDataQuery.data.allCausesJson.edges.forEach(
       ({ node: { path, data } }) => {
+        console.log(data)
```
Comment (amaliwanag2) [inline gatsby-node.esm.js:183]: x

File: src/data/causes/cats.json [hunk 5]
```diff
@@ -21,7 +21,8 @@
       "landing": {
         "title": "Open tabs.  Help shelter cats!",
```
Comment (amaliwanag2) [inline src/data/causes/cats.json:22]: it looks like the copy has changed
https://www.figma.com/file/U69FDY32xoACHgIxyQaGD0/TFC-Brand-Visuals-Exploration?node-id=432%3A736

File: src/data/causes/seas.json [hunk 6]
```diff
@@ -0,0 +1,55 @@
...
+      "ogDescription": "Open browser tabs to clean our seas, for free. It's that easy. Join #teamseas today!",
+      "ogImage": "../../img/cats/1.png",
+      "causeSpecificKeywords": "charity, seas, TeamSeas, teamseas, #teamseas, ocean, oceans, sharks, whales, dolphins, fish, environment, global warming, river, rivers, lake, lakes, pollution, extension, new tab, chrome, help, donation, raise money, money, easy, ways to donate, free, best, home, youtube, safe, volunteer, internet, tab for a cause, impact, legitimate, food, facebook, twitter, reddit, instagram, tumblr"
+    },
+    "sections": {
+      "landing": {
+        "title": "The easiest way to save our seas",
+        "subtitle": "Every tab opened raises money for #TeamSeas, an internet-led movement raising $30 million to clean up 30 million pounds of trash from our oceans, rivers, and lakes.",
+        "ctaImg": "../../img/cats/1.png",
+        "waveColor": "#5094FB"
```
Comment (amaliwanag2) [inline src/data/causes/seas.json:25]: Screen Shot 2021-12-06 at 9 33 40 AM

looks like the wave is missing.  Not exactly sure what's going on, but you'll need to add 'waveColor' to the graphql query in gatsby-node-esm as well.  Here's a tutorial on how the system is working for context: https://www.gatsbyjs.com/docs/tutorial/part-4/
but the tldr is to just specify 'waveColor' under ctaImg on line 157 of gatsby-node.esm.js

File: src/components/Landing.js [hunk 7]
```diff
@@ -0,0 +1,176 @@
...
+                <img
+                  data-test-id="tab-logo-with-text"
+                  src={logoWhite}
+                  style={{ height: 40 }}
+                />
+              </Link>
+            </div>
+          </div>
+
+          <MoneyRaisedDisplay color="inherit" />
```
Comment (amaliwanag2) [inline src/components/Landing.js:121]: you'll want to make it white as it is no longer guaranteed to be inherited.

**Review by amaliwanag2 — CHANGES_REQUESTED** [review]
looks like it's pretty close.  Just need to resolve the wave and a couple small things

**Pull request merged**